frontend/public/components/utils/_label.scss

```diff
@@ -25,6 +25,33 @@
   }
 }
 
+// override label and label group default max widths
+.pf-c-label-group,
+.pf-c-label-group__main,
+.pf-c-label-group__list,
+.pf-c-label-group__list-item,
+.pf-c-label {
+  max-width: 100%;
+}
+
```

Comment:
If this is needed we should scope this to a custom style. We are moving away from overriding PatternFly rules directly.